### DISCUSSION TITLE
Can't resize properly.

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#3743] [Q&A] [answered]
Hello Artf!
I can't resize images neighter columns :/
Check the vídeo, have you or anyone here passed by it?

https://user-images.githubusercontent.com/2287371/131848929-a690f352-f09e-403e-8221-106cd12fcb42.mp4

**Comment by artf** [comment]
Hi @ronaldohoch unfortunately, I can't help here just by looking at the video. Are you able to check from the devtools what is actually happening? From the official demo, I see resizing behaves correctly so, it would be helpful to have a reproducible demo.

**Comment by ronaldohoch** [comment]
I'm trying to put it online, but it has a lot of code to some tools like stackblitz to load, if you allow me, i can put in a private repo and set you as member, i will.

Here are two videos, the first is with the property `'gjs-blocks-basic': { flexGrid:1 }`
The second is with the property `'gjs-blocks-basic': { flexGrid:0 }`

it seems to me that I have to click and drag a lot of the mouse to change a few % of the column or image size. It's not fluid like in the demo.
In the demo I have to move the mouse a lot, but it seems to be 1/10 of what I need to move in my version.


https://user-images.githubusercontent.com/2287371/132520142-052db69a-4f7b-4b4a-9719-7c9b968141e8.mp4


https://user-images.githubusercontent.com/2287371/132520399-a059ab65-34ae-4ff5-9405-cc5aba13fe48.mp4

**Comment by ronaldohoch** [comment]
I've made the bug in https://jsfiddle.net/zdqsgwp3/

What i've made?
- Copied the content of the file in dist folder: https://github.com/artf/grapesjs-blocks-basic/blob/master/dist/grapesjs-blocks-basic.min.js
- Pasted in the js part of fiddle, before the grapesjs.init({})
- Dragged a column block
- Tried to resize it.

https://user-images.githubusercontent.com/2287371/132553936-2e4d2853-b67b-4c4c-8e6a-cd1b0032cebe.mp4

_it's Brave browser, in incognito mode_

**Reply by artf** [reply]
Seems more like an issue with the columns component and the resizing with flex. I don't have time to fix those blocks but you can definitely create your custom components and decide how to handle rows/columns

**Comment by ronaldohoch** [comment]
I was looking into it and i don't know why, but, if i integrate the grapesjs-mjml, it brokes the resizing of whole project :/
I'm trying to find what can cause it.

**Reply by ronaldohoch** [reply]
In mjml plugin we can't resize the image, and i'll keep investigating 
![image](https://user-images.githubusercontent.com/2287371/132686625-5af67158-d412-4cef-946d-967f77ded994.png)

It's from https://grapesjs.com/demo-mjml.html and it's equals to my local

**Reply by artf** [reply]
this is by design https://github.com/artf/grapesjs-mjml/blob/a595f11926b17e978ef0cb75cc4fc82cf9b8cf7b/src/components/Image.js#L15

**Comment by ronaldohoch** [comment, marked as answer]
It has a property at mjml, i could add the resize:true. (https://github.com/artf/grapesjs-mjml/blob/master/src/components/Image.js#L15)
But, it keeps with a strange behavior, it stops on resizing and i need to do it again. 🤔

I'm started to think that it's not a good idea to allow the users to resize manually the e-mail images. 🤔

https://user-images.githubusercontent.com/2287371/132736925-1e0ae041-ab7d-47a1-aa15-5e9911ab59cf.mp4

**Reply by ronaldohoch** [reply]
As artf say, it's by design, so, is just remove it if needed.
